backend/src/apiserver/resource/resource_manager.go

```diff
@@ -41,7 +41,6 @@ import (
 const (
 	defaultPipelineRunnerServiceAccountEnvVar = "DefaultPipelineRunnerServiceAccount"
 	defaultPipelineRunnerServiceAccount       = "pipeline-runner"
-	defaultServiceAccount                     = "default-editor"
```

Comment:
defaultServiceAccount is not necessary, because we can just configure it by `defaultPipelineRunnerServiceAccountEnvVar`